src/components/ContentHighlights/data/hooks.js

```diff
@@ -51,11 +51,19 @@ export function useHighlightSet(highlightSetUUID) {
     }
   }, [highlightSetUUID]);
 
+  const updateHighlightSet = useCallback((highlightSetContentItems) => {
+    setHighlightSet((prevHighlightSet) => ({
+      ...prevHighlightSet,
+      highlightedContent: highlightSetContentItems,
+    }));
+  }, []);
```

Comment:
Why are we using `useCallback` here?

Reply:
Mostly because I was following the example of the `getEnterpriseCuration` method above and the `updateEnterpriseCuration` method in useEnterpriseCuration. Is there a reason why it should not be there?